roles/matrix-backup-borg/templates/config.yaml.j2

```diff
@@ -0,0 +1,37 @@
+#jinja2: lstrip_blocks: "True", trim_blocks: "True"
+
+location:
+  source_directories:
+  - {{ matrix_backup_borg_source }}
```

Comment:
`matrix_backup_borg_source` -> `matrix_backup_borg_source_directories`?

Reply:
It's done with 1 var to mount the whole /matrix dir into container. With multiple vars the human factor will be involved, so you accidentally may add something outside of mounted directory and your dirs will not be backuped.

Are you sure you want to change it?
To be honest, I look more into configuration extensions in the next iteration(-s), like it's done in other roles (eg synapse)

Reply:
Oh, I see! You're right. Adding multiple sources will complicate things for mounting as well.

I think we can have both though?

```yaml
# A list of all directories that will be backed up.
# We only automatically mount `matrix_backup_borg_source_directory` into the borgbackup container.
# Should you override this and add additional directories to `matrix_backup_borg_source_directories`,
# you'll need to ensure these volumes are mounted into the borgbackup container.
# To mount additional volumes there, see `matrix_backup_borg_additional_volumes`.
matrix_backup_borg_source_directories: ["{{ matrix_backup_borg_source_directory }}"]

matrix_backup_borg_source_directory: "/matrix"
```

---------

Alternatively, we can have a `for` loop in the systemd `.service` file. There may be other problems with using multiple source directories though.

If you think this overly complicates things for no good reason, we can keep things as they are.